packages/cdktf-cli/package.json

```diff
@@ -52,7 +52,8 @@
     "stream-buffers": "^3.0.2",
     "strip-ansi": "^6.0.0",
     "uuid": "^8.3.0",
-    "yargs": "^15.1.0"
+    "yargs": "^15.1.0",
+    "zod": "^1.11.7"
```

Comment:
That lib looks nice! Is there a reason to not use v2 or v3?

Reply:
Ah found out why myself: Because v3 requires TypeScript 4.1+ and v2 is discontinued 😄

Reply:
Perhaps we should upgrade Typescript - for the CLI that shouldn't be a problem.